### PR TITLE
Added samples of Shapes, Charts, Table

### PR DESCRIPTION
**Pull request by DharanitharanA**
Added UG documentation for below samples:

1. Shapes 
2. Charts
3. Table

### CODE REVIEW

File: Tables/Insert-column-in-table/.NET/Insert-column-in-table/Insert-column-in-table.csproj [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <RootNamespace>Insert_column_in_table</RootNamespace>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Syncfusion.Presentation.NET" Version="*" />
+    <PackageReference Include="Syncfusion.PresentationRenderer.NET" Version="20.4.0.40" />
```
Comment (Suriya-Balamurugan) [inline Tables/Insert-column-in-table/.NET/Insert-column-in-table/Insert-column-in-table.csproj:13]: Remove Renderer NuGet

File: Tables/Insert-row-in-table/.NET/Insert-row-in-table/Insert-row-in-table.csproj [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <RootNamespace>Insert_row_in_table</RootNamespace>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Syncfusion.Presentation.NET" Version="*" />
+    <PackageReference Include="Syncfusion.PresentationRenderer.NET" Version="20.4.0.40" />
```
Comment (Suriya-Balamurugan) [inline Tables/Insert-row-in-table/.NET/Insert-row-in-table/Insert-row-in-table.csproj:13]: Remove Renderer NuGet

File: Tables/Merge-table-cells/.NET/Merge-table-cells/Merge-table-cells.csproj [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <RootNamespace>Merge-table-cells</RootNamespace>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Syncfusion.Presentation.NET" Version="*" />
+    <PackageReference Include="Syncfusion.PresentationRenderer.NET" Version="20.4.0.40" />
```
Comment (Suriya-Balamurugan) [inline Tables/Merge-table-cells/.NET/Merge-table-cells/Merge-table-cells.csproj:13]: Remove Renderer NuGet

File: Tables/Modify-existing-table/.NET/Modify-existing-table/Modify-existing-table.csproj [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <RootNamespace>Modify_existing_table</RootNamespace>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Syncfusion.Presentation.NET" Version="*" />
+    <PackageReference Include="Syncfusion.PresentationRenderer.NET" Version="20.4.0.40" />
```
Comment (Suriya-Balamurugan) [inline Tables/Modify-existing-table/.NET/Modify-existing-table/Modify-existing-table.csproj:13]: Remove Renderer NuGet

File: Tables/Remove-table/.NET/Remove-table/Remove-table.csproj [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <RootNamespace>Remove_table</RootNamespace>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Syncfusion.Presentation.NET" Version="*" />
+    <PackageReference Include="Syncfusion.PresentationRenderer.NET" Version="20.4.0.40" />
```
Comment (Suriya-Balamurugan) [inline Tables/Remove-table/.NET/Remove-table/Remove-table.csproj:13]: Remove Renderer NuGet

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: Tables/Insert-column-in-table/.NET/Insert-column-in-table/Insert-column-in-table.csproj (hunk 1 above)
Comment (DharanitharanA) [reply Tables/Insert-column-in-table/.NET/Insert-column-in-table/Insert-column-in-table.csproj]: Removed Renderer NuGet.

File: Tables/Insert-row-in-table/.NET/Insert-row-in-table/Insert-row-in-table.csproj (hunk 2 above)
Comment (DharanitharanA) [reply Tables/Insert-row-in-table/.NET/Insert-row-in-table/Insert-row-in-table.csproj]: Removed Renderer NuGet.

File: Tables/Merge-table-cells/.NET/Merge-table-cells/Merge-table-cells.csproj (hunk 3 above)
Comment (DharanitharanA) [reply Tables/Merge-table-cells/.NET/Merge-table-cells/Merge-table-cells.csproj]: Removed Renderer NuGet.

File: Tables/Modify-existing-table/.NET/Modify-existing-table/Modify-existing-table.csproj (hunk 4 above)
Comment (DharanitharanA) [reply Tables/Modify-existing-table/.NET/Modify-existing-table/Modify-existing-table.csproj]: Removed Renderer NuGet.

File: Tables/Remove-table/.NET/Remove-table/Remove-table.csproj (hunk 5 above)
Comment (DharanitharanA) [reply Tables/Remove-table/.NET/Remove-table/Remove-table.csproj]: Removed Renderer NuGet.

**Review by Suriya-Balamurugan — APPROVED** [review]
I don't have any concerns in this code changes.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**